app/src/ingest_la_county_policy.py

```diff
@@ -16,12 +16,20 @@ def _ingest_la_county_policy(
     skip_db: bool = False,
     resume: bool = False,
 ) -> None:
+    common_base_url = "https://epolicy.dpss.lacounty.gov/epolicy/epolicy/server/general/projects_responsive/ePolicyMaster/mergedProjects/"
+
     def prep_json_item(item: dict[str, str]) -> dict[str, str]:
         # More often than not, the h2 heading is better suited as the title
         item["title"] = item["h2"]
+
+        # Include the program name in the document title
+        program_name = item["h1"]
+        item["title"] = f"{program_name}: {item['title']}"
         return item
 
-    common_base_url = "https://epolicy.dpss.lacounty.gov/epolicy/epolicy/server/general/projects_responsive/ePolicyMaster/mergedProjects/"
+    chunking_config = DefaultChunkingConfig()
+    # The document name is the same as item["h2"], so it is redundant to include it in the headings
+    chunking_config.include_doc_name_in_headings = False
```

Comment:
Fixes redundant headings in chunk.headings since the `document.name` = `item["title"]` and a webpage's title is always the level 2 heading `item["h2"]`.